src/main/java/net/atos/zac/util/IndexerenRESTService.java

```diff
@@ -63,4 +64,10 @@ public String indexeren(@PathParam("aantal") int aantal) {
         });
         return info.toString();
     }
+
+    @POST
+    @Path("commit")
```

Comment:
Maybe we can rename this to something a bit more meaningful because I think it is not immediately clear what this method does to people using this API? Commit what? a developer could ask. Committing is a Solr/search engine-specific action but has little meaning in the context of an indexeren service I feel.

Reply:
How would you feel about something like `wait-until-ready` ?

Reply:
Done :-)